index.html

```diff
@@ -364,15 +364,17 @@
 				<h3>Determining the primary pointer</h3>
 				<div>When firing a pointer event, a pointer is considered primary if:
 					<ul>
-						<li>The pointer represents a mouse device.</li>
+						<li>The pointer represents a primary mouse input.</li>
 						<li>The pointer represents a primary touch input.</li>
 						<li>The pointer represents a primary pen input.</li>
 					</ul>
 				</div>
 				<dl>
+					<dt>primary mouse input</dt><dd>A pointer representing mouse input is considered the <i>primary mouse input</i> if it dispatched any pointer events (such as <code>pointerdown</code> or <code>pointermove</code>) when no other active pointers representing a mouse input existed. However, see the note about <a href="#multiple-mouse-inputs">multiple mouse inputs</a>.</dd>
```

Comment:
I think this (and the existing touch/pen sections below when considering hover-capable pointers) isn't quite right.  Eg. consider the case of two mice (or pens) both hovering without either being active.  One needs to be primary, right?  What happens in Edge today when a pen first becomes visibly by the digitizer - does it have `isPrimary=true` from the first `pointermove`, or only after the first `pointerdown` event?

I think the essential properties are that:
 - At any given time exactly one pointer of each type is considered primary
 - When the first pointer of a given type becomes active, that pointer also becomes the primary pointer